.circleci/config.yml

```diff
@@ -326,12 +326,14 @@ commands:
           no_output_timeout: 10m
           command: |
             if [[ -n "${GCLOUD_SERVICE_ACCOUNT_JSON}" ]]; then
+              gcloud firebase test android models list
               gcloud firebase test android run --type robo \
                 --app app/build/outputs/apk/release/app-release.apk \
                 --device model=harpia,version=23,locale=en,orientation=portrait \
                 --device model=blueline,version=28,locale=en,orientation=portrait \
-                --device model=Pixel2,version=30,locale=en,orientation=portrait \
+                --device model=Pixel3,version=30,locale=en,orientation=portrait \
```

Comment:
Drop a note that it's x86 virtual device for future context?

Reply:
I don't know if it's a x86 virtual device, but it's a virtual device, will update the note

Reply:
Suppose we used virtual specifically because it's x86 - to test if that arch builds / runs at all